cddl/eat-tbd-labels.cddl

```diff
@@ -14,3 +14,5 @@ ean-chip-version = <TBD38>
 ean-board-version = <TBD39>
 ean-device-version = <TBD40>
 intended-use = <TBD41>
+manifests = <TBD42>
+swevidence = <TBD43>
```

Comment:
I guess we should also add
```
manifests = 42
swevidence = 43
```
to eat-tbd-labels-validate.cddl

Reply:
Fixed